google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -92,7 +92,7 @@ public OSProcessHandler createProcessHandler(
             }
 
             AppEngineStandardRunTask runTask =
-                new AppEngineStandardRunTask(runConfiguration);
+                new AppEngineStandardRunTask(runConfiguration, programRunner.getRunnerId());
```

Comment:
IJ populates it. It has two ProgramRunners to run Runs and Debugs, and getRunnerId() was the only place I could find that allows us to distringuish between runs and debugs. I used the debugger to check these values.